modules/microshift-manifests-override-paths.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+// * microshift//running_applications/microshift-applications.adoc
+
+:_content-type: PROCEDURE
+[id="microshift-manifests-override-paths_{context}"]
+= Override the list of manifest paths
+You can override the list of default paths with the following procedure.
+
+.Procedure
+
+. You can set `manifests.kustomizePaths` to `++"++/opt/alternate/path++"++` in the configuration file for a single path. Alternatively, you can set `kustomizePaths` to `++"++/opt/alternative/path.d/++*"++` in the configuration file for a glob pattern.
```

Comment:
```suggestion
. You can set `manifests.kustomizePaths` to `++"++/opt/alternate/path++"++` in the configuration file for a single path. Alternatively, you can set `kustomizePaths` to `++"++/opt/alternative/path.d/++*"++` in the configuration file for a glob pattern.
```
I might break this into two bullets under the heading you have as the intro:

. Override the list of default paths by inserting your own values and running one of following commands:

. . Set `manifests.kustomizePaths` to `<++"++/opt/alternate/path++"++>` in the configuration file for a single path.

. .  Set `kustomizePaths` to `++"++</opt/alternative/path.d/>++*"++` in the configuration file for a glob pattern.

I assume you can only do one or the other? If both, we should create the third option.